.rubocop.yml

```diff
@@ -4,7 +4,7 @@ AllCops:
   - vendor/**/*
   - "*/puppet/Puppetfile"
   - "*/puppet/.tmp/**/*"
-  TargetRubyVersion: 1.9
+  TargetRubyVersion: 2.5
```

Comment:
Can we move the rubocop/linting related fixes to the dedicated PR? (we would be able to merge it quickly)

Reply:
I removed this change from the PR.